spec/factories/abilities.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :ability do
```

Comment:
Consider deleting unused code. Do you think we will need this factory in future issues?